presidio-analyzer/setup.py

```diff
@@ -11,8 +11,11 @@
 with open(path.join(this_directory, 'README.MD'), encoding='utf-8') as f:
     long_description = f.read()
 
-with open(os.path.join(parent_directory, "VERSION")) as version_file:
-    __version__ = version_file.read().strip()
+try:
+    with open(os.path.join(parent_directory, "VERSION")) as version_file:
+        __version__ = version_file.read().strip()
+except:  # noqa: E722
+    __version__ = os.environ.get("VERSION", "0.0.1-alpha")
```

Comment:
Consider renaming the env variable to PRESIDIO_VERSION or something similar. Users might have this kind of variable set up before running this.